### PR TITLE
refactor(ui): implement new cloud nav design

### LINKED ISSUE #15843: Replace cloud navigation item with split Application and Account navigation
__Proposal:__
Replace cloud navigation item with split Application and Account navigation:
- Add a top-level navigation bar that houses the account navigation
- Add an ever-present Upgrade button to the top-level navigation bar
- Add a logo to the top-level navigation that acts as a home button
- Remove the logo and cloud from the application navigation bar.

__Current behavior:__
Users are currently unable to find the option to upgrade to a usage-based payment plan or their account settings in the application.

__Desired behavior:__
Users are able to easily find the upgrade option. Users are easily able to find their account settings.

__Alternatives considered:__
- Adding an upgrade button below the existing navigation bar
- Adding the current subscription level next to the org name subtitle with a popover upgrade option
- Adding an ever-present upgrade button next to the org name

__Use case:__
A free teir cloud user is prompted to upgrade and is easily able to find the option.

### PR DESCRIPTION
**Pull request by alexpaxton**
Closes #15843 

Screen Shot 2019-12-11 at 9 59 06 AM

Describe your proposed changes here.

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Comment by mnorriswalker** [comment]
@alexpaxton we need to measure button clicks. Can we add a code snippet to do so? I'm not exactly sure how that's done at Influx. Any tips @desa @russorat ?

**Comment by mnorriswalker** [comment]
We would like to track button clicks once this is released. @russorat I created a Trigger for this in Google Tag Manager tied to the button class name `upgrade-payg--button`

Does this look sound to you? Previewable here:
https://tagmanager.google.com/#/container/accounts/546725706/containers/11229030/workspaces/25

File: ui/src/pageLayout/components/CloudNav.tsx [hunk 1]
```diff
@@ -1,63 +1,101 @@
...
+interface StateProps {
+  orgName: string
+  orgID: string
+}
+
+const CloudNav: FunctionComponent<StateProps> = ({orgName, orgID}) => {
+  const usageURL = `${CLOUD_URL}${CLOUD_USAGE_PATH}`
+  const billingURL = `${CLOUD_URL}${CLOUD_BILLING_PATH}`
+  const handleUpgradeClick = (): void => {
+    window.location.assign(billingURL)
```
Comment (drdelambre) [inline ui/src/pageLayout/components/CloudNav.tsx]: why doesn't the router doesn't take care of this?

Comment (alexpaxton) [reply ui/src/pageLayout/components/CloudNav.tsx]: as in using `router.push`?

Comment (drdelambre) [reply ui/src/pageLayout/components/CloudNav.tsx]: yus

**Comment by alexpaxton** [comment]
Here's the less garbled version of this branch: https://github.com/influxdata/influxdb/pull/16470

**Comment by glinton** [comment]
closed by #16470

**Pull request closed without merging**